heron/tools/cli/src/python/main.py

```diff
@@ -125,7 +127,10 @@ def cleanup(files):
   :return:
   '''
   for cur_file in files:
-    shutil.rmtree(os.path.dirname(cur_file))
+    if os.path.isdir(cur_file):
+      shutil.rmtree(cur_file)
+    else:
+      shutil.rmtree(os.path.dirname(cur_file))
```

Comment:
I follow the desired functionality, just not the need for the if/else. Without it would we not get the desired functionality?